docs/features/techdocs/addons.md

```diff
@@ -52,6 +52,8 @@ Addons are rendered in the order in which they are registered.
 
 ## Installing and using Addons
 
+To start using Addons you need to add the `@backstage/plugin-techdocs-module-addons-contrib` package you your app, you can do that by running this command form the root of your project: `yarn add --cwd packages/app @backstage/plugin-techdocs-module-addons-contrib`
```

Comment:
```suggestion
To start using Addons you need to add the `@backstage/plugin-techdocs-module-addons-contrib` package to your app. You can do that by running this command from the root of your project: `yarn add --cwd packages/app @backstage/plugin-techdocs-module-addons-contrib`
```

Minor typo, might suggest this?

Reply:
Hi @awanlin thanks again for contributing, could you consider this suggestion?

Reply:
Yes, for sure, clearly my wiring skills failed me at the end of the day

Reply:
Ok, all cleaned up